sound/soc/sof/Kconfig

```diff
@@ -240,6 +239,7 @@ config SND_SOC_SOF
 	tristate
 	select SND_SOC_TOPOLOGY
 	select SND_SOC_SOF_NOCODEC if SND_SOC_SOF_NOCODEC_SUPPORT
+	select AUXILIARY_BUS if SND_SOC_SOF_CLIENT
```

Comment:
Still, the SOF_CLIENT should be selecting the AUXILIARY_BUS, not the core.

Reply:
AUXILIARY_BUS is bool as well, it is a driver base stuff.

Reply:
Yes, true, but the code is within the `sof-client.o` object file which is only built if SOF_CLIENT is selected.
I see the SOF_CLIENT as optional feature provided by the core, at least for now. It will change when we move essential code out to clients, but atm we only have debug/developer clients.